### PR TITLE
Added Travis CI integration

### PR DESCRIPTION
**Pull request by mlangbehn**
Added:
- Basic Travis CI configuration
- Basic Textlint configuration
  - The following rules were added:
    - [No Todo](https://github.com/azu/textlint-rule-no-todo)
      - Fails if TODO sections exist
    - [No Empty Section](https://github.com/azu/textlint-rule-no-empty-section)
      - Fails if there is an empty section
    - [Common Misspellings](https://github.com/io-monad/textlint-rule-common-misspellings)
      - Checks for commonly misspelled words
    - [Alex](https://github.com/azu/textlint-rule-alex)
      - Checks for potentially offensive language
    - [Rousseau](https://github.com/azu/textlint-rule-rousseau)
      - Lightweight proofreader

Edits:
- Disabled email notifications from Travis CI
  - Thanks @eggshell for pointing this out

Signed-off-by: Matt Langbehn <mlangbe@us.ibm.com>

### CODE REVIEW

**Comment by eggshell** [comment]
Can you add some explanation as to what the different parts of the textlint config do (e.g. rousseau)?

**Comment by mlangbehn** [comment]
@eggshell, are you looking for comments in the file, a note in the commit message, or in documentation? I would lean towards a documentation note on it.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+cache:
+  directories:
+    - "$(npm config get prefix)/bin"
+
```
Comment (eggshell) [inline .travis.yml:5]:
```
notifications:
  email: False
```

please.

**Comment by eggshell** [comment]
@mlangbehn I was thinking in the commit message, but a documentation note would be good as well. Plus it can change as more things are added (if needed).

File: .travis.yml (hunk 1 above)
Comment (eggshell) [reply .travis.yml:5]: thanks

**Review by eggshell — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (mlangbehn) [reply .travis.yml:5]: Resolved.

**Pull request merged**